src/mailer/templates/user-debt-notification.ts

```diff
@@ -45,7 +45,7 @@ Volgens onze administratie heb je momenteel een schuld bij SudoSOS.
 Het gaat hierbij om een bedrag van:
 ${context.balance.toFormat()}
 
-Ga snel naar de SudoSOS website om je saldo op te hogen! Zo voorkom je dat je een boete krijgt.
+Ga snel naar de SudoSOS website(https://sudosos.gewis.nl) om je saldo op te hogen! Zo voorkom je dat je een boete krijgt.
```

Comment:
Please use the environment variable `URL` instead of hardcoding the url. You can do this by using the `context.url` variable.